test/e2e/scale.go

```diff
@@ -71,7 +73,7 @@ func ScaleToWithin(t *testing.T, scale int, duration time.Duration, latencies La
 
 		wg.Go(func() error {
 			names := test.ResourceNames{
-				Service: test.SubServiceNameForTest(t, fmt.Sprintf("%d", i)),
+				Service: test.SubServiceNameForTest(t, fmt.Sprintf("%[1]*[2]d", width, i)),
```

Comment:
```suggestion
				Service: test.SubServiceNameForTest(t, fmt.Sprintf("%[1]d*%[2]d", width, i)),
```

Reply:
Nah, doesn't look right.

Reply:
https://play.golang.org/p/_ufr5Ni6FPD

Reply:
omg, I totally misunderstood what you were doing.  I didn't even realize that was possible, I thought you were using the index-based format string syntax for some reason.